Makefile

```diff
@@ -62,12 +62,7 @@ packaging/debian/changelog: packaging/debian/changelog.in
 	sed 's/VERSION/$(VERSION)/' $< > $@
 
 $(WEBPACK_TEST): $(COCKPIT_REPO_STAMP) $(shell find src/ -type f) package.json webpack.config.js
-	test/download-dist $${DOWNLOAD_DIST_OPTIONS:-} || \
```

Comment:
Can we also delete `test/download-dist` file, since nothing is now triggering it?

Reply:
Thanks for spotting! Removed.